tsconfig.json

```diff
@@ -20,6 +20,9 @@
     "paths": {
       "@coinbase/build-onchain-apps": [
         "src/index.ts"
+      ],
+      "@coinbase/onchainkit/*": [
+        "node_modules/@coinbase/onchainkit/esm/*"
```

Comment:
Needed to mock the `@coinbase/onchainkit` package in my tests.

Reply:
From where are you running your tests? Why you even have to touch this?